Makefile

```diff
@@ -173,6 +173,11 @@ ko-build-local: test ## Build local container image with ko
 	$(shell (docker version | { ! grep -qi podman; } ) || echo "DOCKER_HOST=unix://$$HOME/.local/share/containers/podman/machine/qemu/podman.sock ") KO_DOCKER_REPO=ko.local ko build -B ./cmd/${CMD_NAME} -t ${IMAGE_TAG}
 	docker tag ko.local/${CMD_NAME}:${IMAGE_TAG} ${IMG}
 
+.PHONY: ko-build-local-bare
+ko-build-local-bare: manifests generate ## Build local container image with ko
```

Comment:
There is a lot of material duplicated here; that is unfortunate. This is the same as `ko-build-local` except that it does not depend on the `test` target, right? I assume that is to avoid a circular dependency among `make` targets? I thought that `make test` was for running _unit_ tests. I think we can avoid circular dependency _and_ avoid duplicating Makefile contents if we distinguish three kinds of tests: unit tests, integration tests, and E2E tests.  See how I did some of that in #1769 .

Reply:
As usual, great comment.  I foolishly didn't realize the make test is selective.  I switched to your suggestion.